lambdas/build-proven-user-identity-details/src/main/java/uk/gov/di/ipv/core/buildprovenuseridentitydetails/domain/ProvenUserIdentityDetails.java

```diff
@@ -9,22 +9,26 @@
 import uk.gov.di.ipv.core.library.domain.Address;
 
 import java.util.List;
+import java.util.Map;
 
 @EqualsAndHashCode(callSuper = false)
 @ExcludeFromGeneratedCoverageReport
 @Data
 @Builder
 public class ProvenUserIdentityDetails {
     @JsonProperty private final String name;
+    @JsonProperty private final Map<String, String> formattedName;
```

Comment:
I'm not sure about `formattedName` here - I think the original `name` is actually the formatted one, and I'm also not sure about having it just be a map.

As below, I wonder about just making this `List<NamePart> nameParts`?